### ISSUE TITLE
EOWG Template Issues

### ISSUE THREAD

**Issue by richnoah** [w3c/apg-redesign#6]
The following is a list of issues discovered with the EOWG Template while developing the ARIA APG Prototype.

- The logo/title on the top left should be a link to the root of the document.
- Lack of W3 favicon
- Nav shouldn’t have aria-label="Context for this document"
- H1 shouldn’t be wrapped in a banner (). Make sure headings are not wrapped in banners
- While tabbing headings are not announcing the name
- It is not clear the footer is a footer. The landmark wasn’t announced by the screen reader.
- Current items in the main nav don’t have the yellow bar underneath them

**Comment by shawna-slh** [comment]
I think a lot of these are addressed in the published version, see https://www.w3.org/WAI/WCAG2/supplemental/

> ... title on the top left should be a link to the root of the document

That would be redundant with the links right around it. I think if would be more cluttering that useful. Happy to learn more otherwise.

> ...others..

If there are remaining issues, it would be most helpful to have the fix submitted as a Pull Request, or an issue in https://github.com/w3c/wai-minimal-header-design/issues

Thanks!
